### DISCUSSION TITLE
API image requests

### DISCUSSION THREAD

**Discussion by AlphaDog01** [mealie-recipes/mealie#2700] [Q&A]
I placed a question a week ago and it was deleted not sure if I did something wrong or not please let me know if I did.

I am looking for suggestions on how to properly get the current image from the meal planner recipe for the day? I have everything else in my api app working except this. The image call comes back with raw characters not an image and I am not sure what to with that to produce the image.
Any help would be great

**Comment by boc-the-git** [comment]
> I placed a question a week ago and it was deleted

It wasn't deleted, I responded.
https://github.com/mealie-recipes/mealie/issues/2689#issuecomment-1783776791

**Reply by AlphaDog01** [reply]
Oh for some reason I don't see that. Thanks though. I was building this in a separate app that is clearly not on the same server so it needs a backend to handle requests to api. I also run behind cloudflare so it makes it harder. I was able to figure this out today the issue was it was not converting to base64 and now it is so I am all set.

For anyone in the future this can be done with any curl request no matter the back end and you just have to process the returned webp into a base64 string then you can display it without issues. Again this only happened because I am behind cloudflare zero trust which blocks all api access from cross domain talk.
